src/pages/Library/Library.tsx

```diff
@@ -24,12 +26,18 @@ const Library = () => {
   const { page } = usePagination();
 
   const { data: bookListData } = useGetBookListQuery({ page, size });
+  const { data: borrowedBookListData } = useGetBookBorrowsQuery({ page: 0, size: MAX_BORROWABLE_BOOKS });
+
   return (
     <div>
       <PageTitle>도서검색</PageTitle>
       <div className="mb-5 flex w-full items-center justify-between">
         <SearchSection />
-        <BorrowStatus librarian={librarian} canBorrow={false} />
+        <BorrowStatus
+          librarian={librarian}
+          borrowedBookCount={borrowedBookListData?.totalElement || 0}
+          MAX_BORROWABLE_BOOKS={MAX_BORROWABLE_BOOKS}
```

Comment:
넘겨주는 값 자체는 대문자 + 스네이크 표기법으로 상수 데이터 네이밍으로 좋은 것 같은데 props명은 정해진 값이 아닌 상위에서 양한 값들을 받아올 수 있는 속성이라 네이밍이 어색한 것 같아요!

```suggestion 
          maxBorrowableBooks={MAX_BORROWABlE-BOOKS}
```
로 변경 어떨까요?

Reply:
아항 넵넵!